documentation-website/Writerside/topics/Getting-Started-with-Exposed.topic

```diff
@@ -58,11 +58,9 @@
 
     </chapter>
     <chapter title="Create a new project" id="create-new-project">
-        <p>
```

Comment:
Does it mean that the problem happens when the tag `<resource>` or `<include>` is added to the `<p>` block?

Reply:
Yes, it seems so!